active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,120 @@
+
+# Background
+
+There’s currently no consistent and Fluent way to address the common UX pattern of a breadcrumb.
+This control is needed for situations where the user's navigation trail (in a file system or menu system) needs
+to be persistently visible.
+A Breadcrumb control is movitated by its use in many app scenarios and supporting developers in migrating from WPF.
+
+This control is similar to an ItemsControl (has an ItemsSource property), but does not derive from it.
+
+## V1/V2 Breadcrumb
+
+The work for Breadcrumb has been scoped to V1 and V2 stages of the API.
+In V1 Breadcrumb no chevrons are interactable to show children.
+V2 Breadcrumb will add functionality to have flyouts from chevrons for every node,
+to view children of that node. This spec focuses on V1 Breadcrumb,
+but keeping in mind that V1 is positioned in a way to have V2 features smoothly added later. 
+
+# API Pages
+
+## Breadcrumb class
+
+Use the Breadcrumb control to private persisting 'trail' that shows the user's navigation path.
+
+This is not the 'history' of the user's navigation, but the direct path of pages or folders from
+the root node to their current position. For situations where resizing means the full path can't be shown,
+the Breadcrumb will 'crumble' starting at the root node. In addition,
+the leaf node can truncated if crumbling from the root node to the leaf node's parent is not enough for a resizing scenario. 
+
+A Breadcrumb control with 3 nodes:
+
+![Breadcrumb default with 3 nodes](images/Breadcrumb_default.PNG)
+
+A Breadcrumb "crumbling" with 3 visible nodes:
+
+![Breadcrumb crumbling with 3 visible nodes](images/Breadcrumb_crumbling.PNG)
+
+A Breadcrumb control with the last node truncated:
+
+![Breadcrumb_crumbled with last node truncated](images/Breadcrumb_truncation.PNG)
+
+The following Xaml markup  creates a simple Breadcrumbe control with three nodes:
+
+![Breadcrumb default with 3 nodes](images/Breadcrumb_default.PNG)
+ ~~~~ 
+     <muxc: Breadcrumb ItemsSource = {"Node", "Node", "Node"} /> 
```

Comment:
Could you recommend a format for the ItemsSource as it is currently just an Object?

Reply:
similar to the ItemsSource in the [ItemsRepeater control](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/items-repeater#:~:text=%20ItemsRepeater%20%201%20Examples.%20If%20you%20have,to%20display%20by%20setting%20the%20ItemsSource...%20More), the value should be of "any type that implements IEnumerable"